test/cron-job.test.ts

```diff
@@ -0,0 +1,104 @@
+import { Testing, ApiObject, Duration, CronOptions, Cron } from 'cdk8s';
+import { Node } from 'constructs';
+import * as kplus from '../src';
+
+test('defaultChild', () => {
+
+  const chart = Testing.chart();
+
+  const cronOptions: CronOptions = {
+    minute: '*',
+    hour: '*',
+    day: '*',
+    month: '*',
+    weekDay: '*',
+  };
+
+  const schedule = new Cron(cronOptions);
+
+  const defaultChild = Node.of(new kplus.CronJob(chart, 'CronJob', {
+    schedule: schedule,
+  })).defaultChild as ApiObject;
+
+  expect(defaultChild.kind).toEqual('CronJob');
+});
+
+test('allow setting jobProperties / default configuration', () => {
+
+  const chart = Testing.chart();
+
+  const schedule = Cron.everyMinute();
+
+  new kplus.CronJob(chart, 'CronJob', {
+    activeDeadline: Duration.seconds(60),
+    backoffLimit: 4,
```

Comment:
No need. This test should just test the required properties and assert against the default values.

Reply:
Makes sense, will update this.